spec/12-the-scala-standard-library.md

```diff
@@ -28,8 +28,8 @@ values which are not implemented as objects in the underlying host
 system.
 
 User-defined Scala classes which do not explicitly inherit from
-`AnyVal` inherit directly or indirectly from `AnyRef`. They can
-not inherit from both `AnyRef` and `AnyVal`.
+`AnyVal` inherit directly or indirectly from `AnyRef`. They can't
+inherit from both `AnyRef` and `AnyVal`.
```

Comment:
"cannot". There is a stray "can't" in section 2.

Reply:
Change to "cannot". What is section 2?